apps/backend/src/authn/gitlab.strategy.ts

```diff
@@ -24,6 +24,7 @@ export class GitlabStrategy extends PassportStrategy(Strategy, 'gitlab') {
     super({
       clientID: configService.get('GITLAB_CLIENTID') || 'disabled',
       clientSecret: configService.get('GITLAB_SECRET') || 'disabled',
+      baseURL: configService.get('GITLAB_BASEURL'),
```

Comment:
Does this still handle the default case properly?

Reply:
It does as the default is set within passport-gitlab2